docs/configuration.rst

```diff
@@ -456,48 +515,60 @@ Map Options
 
     **Default:** ``isometric``
 
-    .. image:: img/map_render_view.png
-       :align: center
-       :alt: Render Views of isometric, topdown and side.
-
     This is the view that your world is rendered from. You can choose from
     different render views:
 
-    ``isometric``
-      A 3D isometric view looking at north-east, north-west, south-west or 
-      south-east (depending on the rotation of the world).
-    ``topdown``
-      A simple 2D top view.
-    ``side``
-      A 2.5D view similar to ``topdown``, but tilted.
+    :Isometric:
+        .. image:: img/map_render_view_isometric.png
 
+        A 3D isometric view with up to 4 rotations.
+    
+    :Topdown:
+        .. image:: img/map_render_view_topdown.png
 
-**Render Mode:** ``render_mode = plain|daylight|nightlight|cave|cavelight``
-	
-    **Default:** ``daylight``
+        A simple 2D top view.
 
-    .. image:: img/map_render_mode.png
-       :align: center
-       :alt: Render Modes of plain, daylight and nightlight, cave and cavelight.
+    :Side:
+        .. image:: img/map_render_view_side.png
+
+        A 2.5D view similar to ``topdown``, but tilted.
 
-    .. image:: img/map_render_mode_cave.png
-       :align: center
-       :alt: Render Modes of plain, daylight and nightlight, cave and cavelight.
+
+**Render Mode:** ``render_mode = daylight|nightlight|plain|cave|cavelight``
+	
+    **Default:** ``daylight``
 
     This is the render mode to use when rendering the world. Possible
     render modes are:
 
-    ``plain``
-        Plain render mode without lighting or other special magic.
-    ``daylight``
-        Renders the world with lighting. This looks better, but takes longer to render.
-    ``nightlight``
-        Like ``daylight``, but renders at night. Hope your world has lots of torches!
-    ``cave``
-        Renders only caves and colors blocks depending on their height 
-        to make them easier to recognize. You can see underground!
-    ``cavelight``
-        A combination of ``cave`` and ``daylight``.
+    :Daylight:      
```

Comment:
The layout and the style of the example images looks good. One more little thing I have about them though. :see_no_evil: 
Did you take the screenshots from low-resolution renders (i.e. low texture size)? The individual blocks seem a little pixelated to me.

Reply:
It's a split between 12 and 16. Generally, I'm using 12 for full map renders, and 16 for my crops - speed and all that. Although all the `render_mode` examples are 16. And then all screenshots get shrunk so they don't take up too much visual space. I can make 16 the standard across the board if you like - or change specific examples.
https://minecraft.ligos.net/worlds/Mapcrafter_Test/config-mapcrafter-test.conf.txt

Reply:
12px renders are totally fine, I thought you might had the resolution lower maybe.

Did you perhaps take the screenshots from a view further zoomed out? Or maybe it's how you cropped / shrunk the screenshots? They look a bit pixelated and somewhat non-dithered to me. Just for example look at the pixels on top of the fountain here:
![image](https://user-images.githubusercontent.com/1788152/47266372-c9c64f80-d535-11e8-97d7-11119aa281e0.png) (I don't see this when I look at different zoom levels on your map)

Reply:
Actually, I think it happens when you zoom completely out and then take the screenshot. Zooming a bit in and shrinking the image (with linear interpolation) should help that.

Reply:
Agreed with the zoom level as the issue. I've seen some "interesting" patterns when you're zoomed out  - mostly with water (see [here](https://minecraft.ligos.net/worlds/Mapcrafter_Test/index.html#MapcrafterTest_Overworld_isometric_daylight/0/1/743/465/64) and below - tweaked to make it really obvious). I'll try them again at max zoom. And tell my paint program to save as 24bit PNGs instead of 8bit palette.

![water-dithering](https://user-images.githubusercontent.com/3079652/47282125-cdd39980-d629-11e8-8361-f3256abf5aae.png)

Reply:
Agreed with the zoom level as the issue. I've seen some "interesting" patterns when you're zoomed out  - mostly with water (see [here](https://minecraft.ligos.net/worlds/Mapcrafter_Test/index.html#MapcrafterTest_Overworld_isometric_daylight/0/1/743/465/64) and below - tweaked to make it really obvious). I'll try them again at max zoom. And tell my paint program to save as 24bit PNGs instead of 8bit palette.

![water-dithering](https://user-images.githubusercontent.com/3079652/47282125-cdd39980-d629-11e8-8361-f3256abf5aae.png)

Reply:
Confirmed: issue was 8bit indexed PNGs. Mapcrafter was set to indexed and my image editor was saving 8 bit PNGs. Changed both to 24bit and redone screenshots - weird pixel patterns are gone! I'll re-do screenshots in 24bit, but will keep Mapcrafter using indexed PNGs (so my poor ADSL connection has some hope of serving them!)